src/components/Theme/src/default-theme.js

```diff
@@ -19,6 +19,18 @@ export default function defaultTheme() {
 		fonts: {
 			baseSize: 16,
 			sizeScale: 1.17,
+			title: {
```

Comment:
I went with `headline`, `title`, `paragraph`, and `label` to be consistent with [Shimaa's doc](https://coda.io/d/ECOM-Website_dM7hmP2aSPk/Design-Specs_suSqm) and Website. Since the names are arbitrary can we just update the eng design to be consistent with Website? It's much easier to update an eng design than updating all of Website.

Sure, we can update `headline` to have the same defaults as `title` or to even point into `title`.

Hmmm, I've reviewed Maker docs and `label` means something different in Maker than it does in Website. In Website `label`s are functionally used as sub-titles. Hmmmm. We may need 5 default text variants.

| Maker | Website |
|-|-|
`headline` | `headline`
`title` | `title`
`paragraph` | `paragraph`
`label` | `ui-font`
no Maker equivalent | `label` (used as subtitles)

We can add a `subtitle` to Maker or Website can just do it themselves, since user-defined variants are supported in beta.

Reply:
My preference is to stick to a definition that's becoming more of a standard for more themeable design systems: https://theme-ui.com/theme-spec

While I often will side with naming conventions that are the same as what we present in our UI, Maker is meant to be used beyond one product for one team. Adopting a larger defined standard helps us more easily make decisions and be interoperable in the future (considering tooling, documentation, and extensibility).

The current product `label` usage is likely to change, I'm not even sure if we're sticking with that name -- it just happens to be a bit more generic than "buttons" and better than "UI Font". It could be used for more things like navigation, controls, or subtitles -- but we haven't mapped that out really. We just know we'd like to provide four options where everywhere `label` is used now has been updated to default to `body`.

Reply:
I had a chat with Shimaa and she helped answer my questions, summary:
1. "label"s within the editor need to be renamed/remapped to "title"s, which is how they're being used already anyway
2. "label"s within Maker just refer to form labels & button text, and should map to paragraph font family and weight (will do in this PR)
3. "body" can be renamed to "paragraph" so there's more alignment / less confusion between Maker and Website

Reply:
i updated the pr after my convos with Shimaa, she's signed off on the designs/theming and has said it's okay to rename "heading" to "title" and "body" to "paragraph" to be more consistent with Website